rotator_pxi/doc/linuxConfig.md

```diff
@@ -0,0 +1,58 @@
+# Linux Configuration
+
+## Kernel Configuration Parameters Selected
+
+*1. CONFIG_E1000E* \
```

Comment:
See above. No need for *1.xxx* (for *) and no need for \. Also those are parameters. I would rather just plantext add them, in ``` - so if you need to copy&paste to kernel config file, you don't have to remove numbers

Reply:
The change of setting will be by GUI. Therefore, it is easier to have the number (1. xxx) here. For the format (for *), fixed in commit: ce7f486. Thanks!